<!doctype html><html><head><title>RNA Strandness</title><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="ie=edge"><link rel=stylesheet href=/css/bootstrap.min.css><link rel=stylesheet href=/css/layouts/main.css><link rel=stylesheet href=/css/navigators/navbar.css><link rel=stylesheet href=/css/plyr.css><link rel=stylesheet href=/css/flag-icon.min.css><link rel=stylesheet href=/css/katex.min.css><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Muli:wght@300;400;500;600"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.min.css><meta property="og:title" content="RNA Strandness"><meta property="og:description" content="Create Time: 2022-09-12 Update Time: 2022-09-12   我一直想要好好的整理一下有关于RNA-seq的实验中，stranded相关信息，但是从我有记录这项任务开始，到今天可能有一年多了。虽说网上已经有了很好的资源，说明的很清楚（参考目录中），但是毕竟是别人的，我觉得还是有必要整理一下，让自己更加清晰。
 首先有很多下游（相对于实验来说）的分析，都需要有这项信息，来获得更加准确的结果，包括但不限于比对、转录组组装等。对于有些资深的人来说，几乎知道实验过程中使用的什么kit，就知道应该使用什么参数，可是仍旧有可能，当我们的数据来自于公共数据库，或者我们的数据是由公司产出，公司并没有使用标准的protocol，而是有一些小的修改，那么就可能造成strand信息的不正确。
 NOTE It is very possible that a sequencing provider/core may make modifications to these kits. For example, in one case we obtained RNAseq data processed with NEBNext Ultra II Directional kit (dUTP method). However instead of using the NEB hairpin adapters, IDT xGen UDI-UMI adapters were substituted, and this results in the insert strandedness being flipped (from RF/fr-firststrand to FR/fr-secondstrand)."><meta property="og:type" content="article"><meta property="og:url" content="https://loganylchen.github.io/zh/posts/rna-seq/transcriptome/rna-strandness/"><meta property="article:published_time" content="2022-09-12T13:53:00+00:00"><meta property="article:modified_time" content="2022-09-12T13:53:00+00:00"><meta name=description content="RNA Strandness"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/styles/atom-one-dark.min.css><link rel=stylesheet href=/css/layouts/single.css><link rel=stylesheet href=/css/navigators/sidebar.css><link rel=stylesheet href=/css/style.css></head><body data-spy=scroll data-target=#TableOfContents data-offset=80><div class="container-fluid bg-dimmed wrapper"><nav class="navbar navbar-expand-xl top-navbar final-navbar shadow"><div class=container><button class="navbar-toggler navbar-light" id=sidebar-toggler type=button onclick=toggleSidebar()>
<span class=navbar-toggler-icon></span></button>
<a class=navbar-brand href=/zh>Logan's Blog</a>
<button class="navbar-toggler navbar-light" id=toc-toggler type=button onclick=toggleTOC()>
<span class=navbar-toggler-icon></span></button><div class="collapse navbar-collapse lang-selector" id=top-nav-items><ul class="navbar-nav ml-auto"></ul></div></div></nav><section class=sidebar-section id=sidebar-section><div class=sidebar-holder><div class=sidebar id=sidebar><form class=mx-auto method=get action=/zh/search><input type=text name=keyword placeholder=Search data-search id=search-box></form><div class=sidebar-tree><ul class=tree id=tree><li id=list-heading><a href=/zh/posts data-filter=all>Posts</a></li><div class=subtree><li><i class="fas fa-plus-circle"></i><a href=/zh/posts/book/>Book</a><ul><li><i class="fas fa-plus-circle"></i><a href=/zh/posts/book/genexii/>GeneXII</a><ul><li><a href=/zh/posts/book/genexii/chapter-1-genes-are-dna-and-encode-rnas-and-polypeptides/ title="Chapter 1 Genes Are DNA and Encode RNAs and Polypeptides">Chapter 1 Genes Are DNA and Encode RNAs and Polypeptides</a></li><li><a href=/zh/posts/book/genexii/chapter-2-methods-in-molecular-biology-and-genetic-engineering/ title="Chapter 2 Methods in Molecular Biology and Genetic Engineering">Chapter 2 Methods in Molecular Biology and Genetic Engineering</a></li></ul></li></ul></li><li><i class="fas fa-plus-circle"></i><a href=/zh/posts/misc/>Misc</a><ul><li><i class="fas fa-plus-circle"></i><a href=/zh/posts/misc/coding-r/>Coding-R</a><ul><li><a href=/zh/posts/misc/coding-r/r-is-not-underscore-friendly/ title="R is not underscore friendly">R is not underscore friendly</a></li></ul></li></ul></li><li><i class="fas fa-plus-circle"></i><a href=/zh/posts/paper/>Paper</a><ul><li><i class="fas fa-plus-circle"></i><a href=/zh/posts/paper/softwares-algorithms/>Softwares-Algorithms</a><ul><li><a href=/zh/posts/paper/softwares-algorithms/hamr-high-throughput-annotation-of-modified-ribonucleotides/ title="HAMR high throughput annotation of modified ribonucleotides">HAMR high throughput annotation of modified ribonucleotides</a></li></ul></li></ul></li><li><i class="fas fa-minus-circle"></i><a class=active href=/zh/posts/rna-seq/>RNA-seq</a><ul class=active><li><i class="fas fa-minus-circle"></i><a class=active href=/zh/posts/rna-seq/transcriptome/>Transcriptome</a><ul class=active><li><a class=active href=/zh/posts/rna-seq/transcriptome/rna-strandness/ title="RNA Strandness">RNA Strandness</a></li></ul></li></ul></li></div></ul></div></div></div></section><section class=content-section id=content-section><div class=content><div class="container p-0 read-area"><div class="hero-area col-sm-12" id=hero-area style=background-image:url(https://loganylchen.github.io/images/post_images/RNA-seq/Transcriptome/RNA-Strandness.jpg)></div><div class=page-content><div class="author-profile ml-auto align-self-lg-center"><img class=rounded-circle src=/images/avatar_hu01b3d05d49046618f204ea6eaefbcdc7_1636650_120x120_fit_box_2.png alt="Author Image"><h5 class=author-name>陈玥茏</h5><p>:date_full</p></div><div class=title><h1>RNA Strandness</h1></div><div class=post-content id=post-content><ul><li><strong>Create Time: 2022-09-12</strong></li><li><strong>Update Time: 2022-09-12</strong></li></ul><blockquote><p>我一直想要好好的整理一下有关于RNA-seq的实验中，stranded相关信息，但是从我有记录这项任务开始，到今天可能有一年多了。虽说网上已经有了很好的资源，说明的很清楚（参考目录中），但是毕竟是别人的，我觉得还是有必要整理一下，让自己更加清晰。</p></blockquote><p>首先有很多下游（相对于实验来说）的分析，都需要有这项信息，来获得更加准确的结果，包括但不限于比对、转录组组装等。对于有些资深的人来说，几乎知道实验过程中使用的什么kit，就知道应该使用什么参数，可是仍旧有可能，当我们的数据来自于公共数据库，或者我们的数据是由公司产出，公司并没有使用标准的protocol，而是有一些小的修改，那么就可能造成strand信息的不正确。</p><blockquote><p><em>NOTE</em>
It is very possible that a sequencing provider/core may make modifications to these kits. For example, in one case we obtained RNAseq data processed with NEBNext Ultra II Directional kit (dUTP method). However instead of using the NEB hairpin adapters, IDT xGen UDI-UMI adapters were substituted, and this results in the insert strandedness being flipped
 (from RF/fr-firststrand to FR/fr-secondstrand). Because this level of detail is not always provided it is highly recommended to confirm your data’s strandedness empirically.</p></blockquote><p>所以最好是能够自己检查数据，并且知道如何通过检查的结果来选择后续的分析参数，以及为什么是这样的选择。</p><h2 id=分类>分类</h2><p>对于NGS来说，分类就非常的简单，只有三种：</p><ul><li><p>read1的方向与transcript相同【forward】，read2的方向与transcript相反【reverse】（第一种）</p></li><li><p>read1的方向与transcript相反【reverse】，read2的方向与transcript相同【forward】（第二种）</p></li><li><p>read1的方向有时与transcript相同，有时相反（第三种）</p></li></ul><p>上述括号中也就是的单词也就是最常作为参数的单词比如：</p><ul><li><p>RF=read1 reverse + read2 forward （第二种）</p></li><li><p>FR=read1 forward + read2 reverse （第一种）</p></li><li><p>F2R1 = read2 forward + read1 reverse (第二种)</p></li><li><p>F1R2 = read1 forward + read2 reverse （第一种）</p></li></ul><p>也就是没有标明数字的，那么前一个指代read1，后一个指代read2，对于标明了数字的，数字前面的是表示该read的方向。</p><p>其中比较需要注意的是<code>TopHat</code>的参数，<code>fr-firststrand</code>（第二种）和<code>fr-secondstrand</code>（第一种）。这里我不太好理解，所以去查了一下tophat的官方文档的说明，说明如下：</p><p>| |
| |
| |
| |</p><p>这里我copy一份别人整理好的表格：</p><p>| |
| |
| |
| |
| |
| |
| |
| |
| |
| |
| |
| |
| |
| |
| |
| |
| |
| |</p><p>方便之后自己再做参考使用。</p><h2 id=kit-and-strandness>Kit and Strandness</h2><p>【待续】</p><h2 id=参考>参考</h2><p><img src=https://rnabio.org/module-09-appendix/0009/12/01/StrandSettings/ alt=https://rnabio.org/module-09-appendix/0009/12/01/StrandSettings/></p></div><div class="row pl-3 pr-3"><div class="col-md-6 share-buttons"></div><div class="col-md-6 btn-improve-page"><a href=https://github.com/loganylchen/loganylchen.github.io/edit/main/content/posts/RNA-seq/Transcriptome/RNA-Strandness.zh.md title="Improve this page" target=_blank rel=noopener><i class="fas fa-code-branch"></i>Improve this page</a></div></div><hr><div class="row next-prev-navigator"><div class="col-md-6 previous-article"><a href=/zh/posts/paper/softwares-algorithms/hamr-high-throughput-annotation-of-modified-ribonucleotides/ title="HAMR high throughput annotation of modified ribonucleotides" class="btn btn-outline-info"><div><i class="fas fa-chevron-circle-left"></i>Prev</div><div class=next-prev-text>HAMR high throughput annotation of modified ribonucleotides</div></a></div></div><hr></div></div></div><a id=scroll-to-top class=btn><i class="fas fa-chevron-circle-up"></i></a></section><section class=toc-section id=toc-section><div class=toc-holder><h5 class="text-center pl-3">Table of Contents</h5><hr><div class=toc><nav id=TableOfContents><ul><li><ul><li><a href=#分类>分类</a></li><li><a href=#kit-and-strandness>Kit and Strandness</a></li><li><a href=#参考>参考</a></li></ul></li></ul></nav></div></div></section></div><footer id=footer class="container-fluid text-center align-content-center footer pb-2"><div class="container pt-5"><div class="row text-left"><div class="col-md-4 col-sm-12"><h5>Navigation</h5><ul><li class=nav-item><a class=smooth-scroll href=https://loganylchen.github.io/zh/#about>关于</a></li><li class=nav-item><a class=smooth-scroll href=https://loganylchen.github.io/zh/#skills>技能</a></li><li class=nav-item><a class=smooth-scroll href=https://loganylchen.github.io/zh/#experiences>经历</a></li><li class=nav-item><a class=smooth-scroll href=https://loganylchen.github.io/zh/#projects>项目</a></li><li class=nav-item><a class=smooth-scroll href=https://loganylchen.github.io/zh/#recent-posts>最近发布</a></li><li class=nav-item><a class=smooth-scroll href=https://loganylchen.github.io/zh/#achievements>成就</a></li></ul></div><div class="col-md-4 col-sm-12"><h5>Contact me:</h5><ul><li><a href=mailto:loganylchen@gmail.com target=_blank rel=noopener><span><i class="fas fa-envelope"></i></span><span>loganylchen@gmail.com</span></a></li><li><span><i class="fas fa-phone-alt"></i></span><span>暂时不公开吧</span></li></ul></div><div class="col-md-4 col-sm-12"><p>Stay up to date with email notification</p><form method=post action=https://blogtrottr.com><div class=form-group><input type=email class=form-control name=btr_email placeholder="Enter email"><br><input type=hidden name=btr_url value=https://loganylchen.github.io/zh/index.xml>
<input type=hidden name=schedule_type value=1>
<small id=emailHelp class="form-text text-muted">By entering your email address, you agree to receive the newsletter of this website.</small>
<button type=submit class="btn btn-info"> Submit</button></div></form></div></div></div><hr><div class=container><div class="row text-left"><div class=col-md-4><a id=theme href=https://github.com/hossainemruz/toha target=_blank rel=noopener><img src=/images/theme-logo_hu8376fd15465fef26ffe66b6bcf0ca686_13669_32x0_resize_box_2.png alt="Toha Theme Logo">
Toha</a></div><div class="col-md-4 text-center">©2022 Copyright.</div><div class="col-md-4 text-right"><a id=hugo href=https://gohugo.io/ target=_blank rel=noopener>Powered by
<img src=/images/hugo-logo.svg alt="Hugo Logo" height=18></a></div></div></div></footer><script type=text/javascript src=/js/jquery-3.4.1.min.js></script><script type=text/javascript src=/js/popper.min.js></script><script type=text/javascript src=/js/bootstrap.min.js></script><script type=text/javascript src=/js/navbar.js></script><script type=text/javascript src=/js/plyr.js></script><script type=text/javascript src=/js/main.js></script><script type=text/javascript defer src=/js/katex.min.js></script><script type=text/javascript defer src=/js/auto-render.min.js onload=renderMathInElement(document.body);></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.18.1/highlight.min.js></script><script src=/js/single.js></script><script>hljs.initHighlightingOnLoad();</script></body></html>